examples/inspection/plot_linear_model_coefficient_interpretation.py

```diff
@@ -121,7 +123,7 @@
 # Our pre-processor will
 #
 # - one-hot encode (i.e., generate a column by category) the categorical
-#   columns;
+#   columns, only for non-binary categorical variables;
```

Comment:
I think that we really wanted to avoid the anti-correlation for the binary feature. It could be argued that we should do the same for the non-binary feature as well but I would not touch it here.

Reply:
This may deserve its own issue.
The story we tell is ambivalent. On one side, we say to use OHE without dropping a reference level for symmetric penalization, on the other side, we recommend to drop one level for binary features.
Eq. (3.12) ff of https://doi.org/10.1093/biostatistics/kxm010 explains why dropping is fine.

As said, I better open an issue for this.

Reply:
agreed